code/index.html

```diff
@@ -2,21 +2,85 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quarantine Times</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
+      </head>
+      <body>
+
+<header class="header1">
+  <div class="hamburger">
+    <div></div>
+    <div></div>
+    <div></div>
+    <div></div>
+  </div>
+
+    <img class="logo-pic" src="qllogo.png" alt="logo">
+    <h1 class="qtimes">Quarantine Times</h1>
+  <nav>
+    <a href="">World</a> |
+    <a href="">Politics</a> |
+    <a href="">Science</a> |
+    <a href="">Style</a> |
+    <a href="">Daily</a>
+  </nav>
+</header>
+       
+<section class="big-news-section">
+<!-- The big news content -->
+    <img src="kneading.jpg" alt="kneadingbread">
+  <div>
```

Comment:
Same with these two, they are siblings. Take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you want to recap about indentation :)